src/utils/apiUtils/chartData.js

```diff
@@ -153,40 +152,40 @@ const getDatesAndPrices = dailyData => {
     });
   });
   return datesAndPrices;
-};
+}
 
 // NOTE: this date has timezone UTC, which is incorrect but works in this
 // case because we are just extracting the time
-const getAdjustedDateForTimestamp = (gmtoffset, timestamp) => {
+function getAdjustedDateForTimestamp(gmtoffset, timestamp) {
   const adjustedTimestamp =
     (timestamp + gmtoffset) * MILLISECONDS_PER_SECOND;
   return new Date(adjustedTimestamp);
-};
+}
 
-const getDateAndTime = (gmtoffset, timestamp, dateAndTimeFormat) => {
+function getDateAndTime(gmtoffset, timestamp, dateAndTimeFormat) {
   const dateAndTime = getAdjustedDateForTimestamp(gmtoffset, timestamp);
   return dateFormat(dateAndTime, dateAndTimeFormat, true);
-};
+}
 
-const getStartOfDayTimestampIndex = (dayIndex, timestampsPerDay) =>
-  Math.floor(dayIndex * timestampsPerDay);
+function getStartOfDayTimestampIndex(dayIndex, timestampsPerDay) {
+  return Math.floor(dayIndex * timestampsPerDay);
+}
 
-const getEndOfDayTimestampIndex = (dayIndex, timestampsPerDay) =>
-  Math.floor((dayIndex + 1) * timestampsPerDay);
+function getEndOfDayTimestampIndex(dayIndex, timestampsPerDay) {
+  return Math.floor((dayIndex + 1) * timestampsPerDay);
+}
 
-const getDatesAndTimesForOneDay = (
+function getDatesAndTimesForOneDay(
   close,
   dayIndex,
   gmtoffset,
   numberOfDays,
+  dateAndTimeFormat,
   timestamp,
   timestampIntervals
-) => {
+) {
   let datesTimesAndPrices = [];
   const timestampsPerDay = timestamp.length / numberOfDays;
-  const dateAndTimeFormat = (numberOfDays === ONE_DAY)
```

Comment:
Since this line will probably result in a bug for any numberOfDays !== (1 or 5), I decided the `dateAndTimeFormat` should be passed into getDatesAndTimesForOneDay.  I'm still not 100% happy with this, since there's a lot of passing around the `dateAndTimeFormat` between functions, but I think this change is an improvement.